projects/sunbird-player-sdk/src/lib/player-utils/service/error/error.service.ts

```diff
@@ -19,4 +20,11 @@ export class ContentCompabilityService {
       return { error: null, isCompitable: true };
     }
   }
+
+  internetConnectivity() {
```

Comment:
method name should be **getInternetConnectivityError()** and also check internet connectivity check here only return error object or null if online